var/spack/repos/builtin/packages/py-python-multipart/package.py

```diff
@@ -15,8 +15,17 @@ class PyPythonMultipart(PythonPackage):
 
     license("Apache-2.0")
 
+    version("0.0.17", sha256="41330d831cae6e2f22902704ead2826ea038d0419530eadff3ea80175aec5538")
     version("0.0.5", sha256="f7bb5f611fc600d15fa47b3974c8aa16e93724513b49b5f95c81e6624c83fa43")
 
     depends_on("py-setuptools", type="build")
```

Comment:
See `python-multipart-0.0.6` switched to `pyproject.toml` and uses `hatchling` for the build:
https://github.com/Kludex/python-multipart/blob/0.0.6/pyproject.toml#L2 

Thus it needs to be added as a build dependency. Otherwise, my build gets:
```py
ModuleNotFoundError: No module named 'hatchling'

Removed file://.stage/spack-stage-py-python-multipart-0.0.17-gguym7ptotjiimfp5uh3vjx6qabbtaxk/spack-src from build tracker '/tmp/pip-build-tracker-_hsbu8pe'
Removed build tracker: '/tmp/pip-build-tracker-_hsbu8pe'
==> Error: ProcessError: Command exited with status 2:
    'opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/python-venv-1.0-xiqalry5slnudksuvgbqdz3yc4tabea6/bin/python3' '-m' 'pip' '-vvv' '--no-input' '--no-cache-dir' '--disable-pip-version-check' 'install' '--no-deps' '--ignore-installed' '--no-build-isolation' '--no-warn-script-location' '--no-index' '--prefix=opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/py-python-multipart-0.0.17-gguym7ptotjiimfp5uh3vjx6qabbtaxk' '.'
See build log for details:
  .stage/spack-stage-py-python-multipart-0.0.17-gguym7ptotjiimfp5uh3vjx6qabbtaxk/spack-build-out.txt

==> Error: Terminating after first install failure: ProcessError: Command exited with status 2:
    'opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/python-venv-1.0-xiqalry5slnudksuvgbqdz3yc4tabea6/bin/python3' '-m' 'pip' '-vvv' '--no-input' '--no-cache-dir' '--disable-pip-version-check' 'install' '--no-deps' '--ignore-installed' '--no-build-isolation' '--no-warn-script-location' '--no-index' '--prefix=opt/spack/linux-ubuntu22.04-skylake/gcc-13.1.0/py-python-multipart-0.0.17-gguym7ptotjiimfp5uh3vjx6qabbtaxk' '.'

------------------------- FAILED py-python-multipart@=0.0.17 -------------------------
```

```suggestion
    depends_on("py-setuptools", type="build")
    depends_on("py-hatchling", type="build", when="@0.0.6:")
```

Reply:
Also remove py-setuptools for 0.0.6 and higher. 
```suggestion
    depends_on("py-setuptools", type="build", when="@:0.0.5")
    depends_on("py-hatchling", type="build", when="@0.0.6:")
```